### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by gamwe6**

### CODE REVIEW

**Comment by gamwe6** [comment]
I signed the CLA

File: how-does-dtp-work.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ DTP comprises three main components:
 Data Models are the canonical formats that establish a common understanding of how to transfer data.
 Adapters provide a method for converting each Provider’s proprietary data and authentication formats into a form that is usable by the system.
 Task Management Library provides the plumbing to power the system.
-Data Models
```
Comment (seehamrun) [inline how-does-dtp-work.md]: Actually I think the right fix for this is to make it a header

### Data Models 

And the above should be a list (adapters and task management have a section below on lines 21 and 31)

Comment (gamwe6) [reply how-does-dtp-work.md]: Thank you @seehamrun for the review

**Comment by seehamrun** [comment]
Hiya, can you check that you've signed the google CLA so we can merge this?

**Comment by seehamrun** [comment]
Due to inactivity I've copied these changes into #831

**Pull request closed without merging**